packages/neuron-ui/src/components/HardwareSign/hd-wallet-sign.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { useCallback, useState, useMemo } from 'react'
```

Comment:
Follow the [Naming Convention]()https://github.com/airbnb/javascript/tree/master/react#naming